common/lib/xmodule/xmodule/js/src/video/09_video_caption.js

```diff
@@ -536,10 +529,7 @@
                         } else {
                             if (state.isTouch) {
                                 self.subtitlesEl.find('.subtitles-menu').html(
-                                    gettext(
-                                        '<li>Transcript will be displayed when ' +
-                                        'you start playing the video.</li>'
-                                    )
+                                    gettext('<li>Transcript will be displayed when you start playing the video.</li>')
```

Comment:
Good catch @clytwynec. We should update this to follow the safe templates best practice: https://openedx.atlassian.net/wiki/display/TNL/Safe+Template+Marathon